flocker/node/agents/blockdevice.py

```diff
@@ -143,10 +143,11 @@ class UnattachedVolume(VolumeException):
     u"The size of the underlying block device."
 )
 
-BLOCK_DEVICE_HOST = Field(
-    u"block_device_host",
+BLOCK_DEVICE_STORAGE_BACKEND_ID = Field(
```

Comment:
storage_backend_id is slightly confusing since we are actually referring to the id of the compute instance the block device is attached to. 'compute_instance_id' might communicate better. i see that the name has already been changed once, so, no need to waste time doing yet another search-replace. If it sounds reasonable, am happy to make the name change in my EBS attach-volume branch.